pkg/sso/client_test.go

```diff
@@ -168,7 +168,7 @@ func profileTestHandler(w http.ResponseWriter, r *http.Request) {
 		Profile Profile `json:"profile"`
 	}{
 		Profile: Profile{
-			ID:             "proj_123",
+			ID:             "client_123",
```

Comment:
Nit: I think it's originally a typo in the old test. Since this is a profile, maybe we should do `profile_123`.